metadata-models/src/main/pegasus/com/linkedin/dataset/DatasetProfile.pdl

```diff
@@ -10,14 +10,32 @@ import com.linkedin.timeseries.TimeseriesAspectBase
   "type": "timeseries",
 }
 record DatasetProfile includes TimeseriesAspectBase {
+  /**
+   * The total number of rows
+   */
+  @Searchable = {
+    "fieldType": "COUNT"
+  }
 	rowCount: optional long
 
+  /**
+   * The total number of columns (or schema fields)
+   */
+  @Searchable = {
+    "fieldType": "COUNT"
+  }
 	columnCount: optional long
 
+  /**
+   * Profiles for each column (or schema field)
+   */
 	fieldProfiles: optional array[DatasetFieldProfile]
 
 	/**
 	* Storage size in bytes
 	*/
+  @Searchable = {
+    "fieldType": "COUNT"
+  }
 	sizeInBytes: optional long
```

Comment:
is it just the diff view or does the indentation of your additions seem off here?

Reply:
whoa no this seems off